### PR TITLE
Testing refactor

### PR DESCRIPTION
**Pull request by alukach**
- **Simplified testing suite**: It seemed strange to have `expect.js`, `happen.js`, and `sinon.js` stored in the tests (rather than stored in `package.json`).  Additionally, it seemed that `karma.conf.js` could be simplified by letting the `files` attribute do most of the work.
- **Disabled output on certain tests**
- **Added tests for `Menu`**

### CODE REVIEW

File: spec/suites/L.DNC.DropZone.FileReader.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ describe("L.DNC.DropZone.FileReader > ", function () {
         var fileReader = null;
 
         beforeEach(function () {
+            console.debug = function(){};
```
Comment (alukach) [inline spec/suites/L.DNC.DropZone.FileReader.js]: This should be unmocked after tests.

**Comment by alukach** [comment]
I believe this is ready for merge. Curious to hear what @thebigspoon thinks of these changes since he set up the initial test framework.

File: spec/karma.conf.js [hunk 2]
```diff
@@ -1,44 +1,22 @@
...
-            'karma-chrome-launcher'],
-
         // frameworks to use
-        frameworks: ['mocha'],
+        frameworks: ['mocha', 'chai', 'sinon'],
 
         // list of files / patterns to load in the browser
-        files: files,
+        files: [
+            "node_modules/leaflet/dist/leaflet.js",
```
Comment (alukach) [inline spec/karma.conf.js]: I feel like this should be `mapbox.js` as that's what we're actually using in prod.  Differences seem to be minimal, but would prefer to keep tests as close to production as possible.

**Comment by alukach** [comment]
_poke_ :point_right: @svmatthews 

Any thoughts on this? I think that this is at a state to be merged in.  I've been building on top of it in another branch, but want to make sure that this is kosher before I rely on it too much.

**Comment by thebigspoon** [comment]
@alukach nice cleanup work! merge that puppy

**Comment by svmatthews** [comment]
Sorry for the no comment, @alukach. Wanted @thebigspoon to take a look!

**Pull request merged**